install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -0,0 +1,273 @@
+|openshift_openstack_num_cns | Number of container native storage nodes to deploy
+|openshift_openstack_num_nodes | Number of application nodes to deploy
+|openshift_openstack_master_flavor| Name of the OpenStack flavor used for master instances
+|openshift_openstack_default_flavor| Name of the Openstack flavor used for all instances, if specific flavor not specified.
+|openshift_openstack_use_lbaas_load_balancer | Boolean value enabling Octavia load balancer (Octavia must be installed)
+|openshift_openstack_docker_volume_size | Minimum size of the Docker volume (required variable)
+|openshift_openstack_external_nsupdate_keys | Updating the DNS with the instance IP addresses
+|ansible_user| Ansible user used to deploy {product-title}. "openshift" is the required name and must not be changed.
+|openshift_openstack_disable_root| Boolean value that disables root access
+|openshift_openstack_user| OCP instances created with this user
+|===
+
+
+[[osev3]]
+==== OSEv3.yml
+
+The OSEv3.yml file specificies all the different parameters and customizations
+relating the installation of OpenShift. 
+
+Below is a condensed version of the file with all required variables for a
+successful deployment. Additional variables may be required depending on what
+customization is required for your specific {product-title} deployment. 
+
+
+[subs=+quotes]
+----
+*$ cat ~/inventory/group_vars/OSEv3.yml*
+---
+
+openshift_deployment_type: openshift-enterprise
+openshift_release: v3.10
+
+openshift_master_default_subdomain: "apps.{{ (openshift_openstack_clusterid|trim == '') | ternary(openshift_openstack_public_dns_domain, openshift_openstack_clusterid + '.' + openshift_openstack_public_dns_domain) }}"
+
+openshift_master_cluster_public_hostname: "console.{{ (openshift_openstack_clusterid|trim == '') | ternary(openshift_openstack_public_dns_domain, openshift_openstack_clusterid + '.' + openshift_openstack_public_dns_domain) }}"
+
+*#OpenStack Credentials:*
+openshift_cloudprovider_kind: openstack
+openshift_cloudprovider_openstack_auth_url: "{{ lookup('env','OS_AUTH_URL') }}"
+openshift_cloudprovider_openstack_username: "{{ lookup('env','OS_USERNAME') }}"
+openshift_cloudprovider_openstack_password: "{{ lookup('env','OS_PASSWORD') }}"
+openshift_cloudprovider_openstack_tenant_name: "{{ lookup('env','OS_PROJECT_NAME') }}"
+openshift_cloudprovider_openstack_blockstorage_version: v2
+openshift_cloudprovider_openstack_domain_name: "{{ lookup('env','OS_USER_DOMAIN_NAME') }}"
+
+*## Use Cinder volume for Openshift registry:*
+openshift_hosted_registry_storage_kind: openstack
+openshift_hosted_registry_storage_access_modes: ['ReadWriteOnce']
+openshift_hosted_registry_storage_openstack_filesystem: xfs
+openshift_hosted_registry_storage_volume_size: 30Gi
+
+
+openshift_hosted_registry_storage_openstack_volumeID: d65209f0-9061-4cd8-8827-ae6e2253a18d
+openshift_hostname_check: false
+ansible_become: true
+
+*#Setting SDN (defaults to ovs-networkpolicy) not part of OSEv3.yml*
```

Comment:
Usually, lines from 178 to 213 are configured in inventory file not in group_vars/OSEv3.yml

Reply:
I'm going to disagree here as metrics, logging, networkpolicy are all part of OSE tunables and should reside within this yaml not inventory.

Reply:
Yes, we can keep them in group_vars/OSEv3.yml